apps/privacy-panel/js/ala/exceptions.js

```diff
@@ -130,7 +132,12 @@ function(panels, BlurSlider, appList, SettingsListener, SettingsHelper) {
 
       if (itemData.type) {
         var type = document.createElement('small');
-        type.textContent = itemData.type;
+        type.setAttribute('data-l10n-id', itemData.type);
+
+        if (itemData.typeArg) {
+          type.setAttribute('data-l10n-args', JSON.stringify(itemData.typeArg));
+        }
```

Comment:
you can use navigator.mozL10n.setAttributes(type, itemData.type, itemData.typeArg);